implants/Cargo.toml

```diff
@@ -29,6 +32,7 @@ itertools = "0.10"
 lsp-types = "0.93.0"
 log = "0.4.20"
 md5 = "0.7.0"
+mockall = "0.12.1"
```

Comment:
optional?

Reply:
Or dev dep?

Reply:
These are not things we define in the workspace, these are things we define where the dependency is actually used (e.g. there is no such thing as workspace `dev-dependencies`)